packages-internal/test-utils/src/createRenderer.tsx

```diff
@@ -740,5 +740,5 @@ function act<T>(callback: () => void | T | Promise<T>) {
 const bodyBoundQueries = within(document.body, { ...queries, ...customQueries });
 
 export * from '@testing-library/react/pure';
-export { act, cleanup, fireEvent };
-export const screen: Screen = { ...rtlScreen, ...bodyBoundQueries };
+export { act, fireEvent };
```

Comment:
`cleanup` is already exported on L742 above.
I don't see the need to duplicate/override the export. 🤷